fastplotlib/graphics/selectors/_base_selector.py

```diff
@@ -16,9 +16,14 @@ class MoveInfo:
     stores move info for a WorldObject
     """
 
-    # last position for an edge, fill, or vertex in world coordinates
-    # can be None, such as key events
-    last_position: Union[np.ndarray, None]
+    # The initial selection. Differs per type of selector
+    start_selection: Any
+
+    # The initial world position of the cursor
+    start_position: Union[np.ndarray, None]
```

Comment:
The `Union` is lingering from python 3.9 before we had `|`

Reply:
fixed!